CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Change Log
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+<a name="0.0.61"></a>
+
+## 0.0.61 (2018-02-27)
```

Comment:
Why? Refactors don't bump versions, https://semver.org/.

Reply:
Sure but these changes weren't from 0.0.61 anyways right? Also changelogs are nice but I still think we are a little early since we really don't even have a stable v1 yet

Reply:
Yeah I'm pretty sure those changes were included in that release.

Reply:
nvm i see whats going on